definitions.json

```diff
@@ -286,6 +306,16 @@
           }
         ],
         "type": "DOT"
+      },
+      "dotsToBtc": {
+        "description": "DOT to BTC conversion rate",
+        "params": [
+          {
+            "name": "amount",
+            "type": "DOT"
```

Comment:
We should change the types in the `exchangeRateOracle` rpc to `BalanceWrapper`, to match the parachain